include/aws/io/private/event_loop_impl.h

```diff
@@ -152,6 +152,16 @@ AWS_IO_API
 struct _OVERLAPPED *aws_overlapped_to_windows_overlapped(struct aws_overlapped *overlapped);
 #endif /* AWS_ENABLE_IO_COMPLETION_PORTS */
 
+/**
+ * @internal - Don't use outside of testing.
+ *
+ * Return the default event loop type. If the return value is `AWS_ELT_PLATFORM_DEFAULT`, the function failed to
```

Comment:
I think you changed this to `AWS_EVENT_LOOP_PLATFORM_DEFAULT` as `AWS_ELT_PLATFORM_DEFAULT` doesn't exist.

Also, it looks like there is nothing returned on a failure to retrieve the default type value. I think its a compile time error?